### DISCUSSION TITLE
TOTP has been extracted to @epic-web/totp

### DISCUSSION THREAD

**Discussion by kentcdodds** [epicweb-dev/epic-stack#361] [Announcements]
Introducing [@epic-web/totp](https://github.com/epicweb-dev/totp)!

Here's the commit that removes the one that was built into the Epic Stack in favor of the new package: https://github.com/epicweb-dev/epic-stack/commit/de9b06d7cce72c68e7b93dc64499f4168b6b96cf

Enjoy!

**Comment by DigitalGoldfish** [comment]
I do now have this warning when building/starting dev mode:

```
 warn  esm-only package: @epic-web/totp
┃ @epic-web/totp is possibly an ESM-only package.
┃ To bundle it with your server, include it in `serverDependenciesToBundle`
┃ -> https://remix.run/docs/en/main/file-conventions/remix-config#serverdependenciestobundle
┗
```
Is that expected and can it be ignored? Does it need to be added to `serverDependenciesToBundle`?

**Reply by kentcdodds** [reply]
That's a false error and will be removed in Remix v2. Everything is fine because epic stack is shipping native ESM
